grpc/server/src/main/java/io/helidon/grpc/server/ServiceDescriptor.java

```diff
@@ -131,6 +131,22 @@ public Descriptors.FileDescriptor proto() {
         return proto;
     }
 
+    /**
+     * Returns the service name prefixed with package directive if one exists.
+     */
+    String getFullName() {
+        return getPackagedName(proto, name);
+    }
+
+    static String getPackagedName(Descriptors.FileDescriptor proto, String name) {
```

Comment:
Why is this a static method? It looks to me like this logic really belongs to the `getFullName` method above (which should probably be public as well), and there is no need for a separate, static method at all.